### ISSUE TITLE
Strategy decision making - convert to ChartIQ

### ISSUE THREAD

**Issue by kenkunz** [tradingstrategy-ai/frontend#358]
# Goal

Improve usability, visual style, and interaction consistency of Strategy decision making chart.

## Details

The current Decision Making chart (example below) is rendered server-side. It lacks interactivity, doesn't match the visual style of our other charts, and is not very mobile-friendly. This should be converted to a ChartIQ chart that is rendered client-side.

<img width="864" alt="image" src="https://user-images.githubusercontent.com/35901/218874354-34b900cc-cb75-4cba-a0b1-86828c8df655.png">

**Comment by AlexTheLion123** [comment]
We also need to come up with way to do this with multiple pairs

**Comment by kenkunz** [comment]
> We also need to come up with way to do this with multiple pairs

Agreed! Let's start with the simpler single pair use case. Then after this has been migrated and implemented with ChartIQ, we can explore the best way to add support for multiple pairs.

**Comment by AlexTheLion123** [comment]
> > We also need to come up with way to do this with multiple pairs
> 
> Agreed! Let's start with the simpler single pair use case. Then after this has been migrated and implemented with ChartIQ, we can explore the best way to add support for multiple pairs.
> 

Cool, actually we just decided to show charts for each individual pair decision making, even if there are a lot of pairs.
